cmd/ci-operator/main.go

```diff
@@ -485,6 +487,11 @@ func (o *options) Run() error {
 			time.Sleep(time.Second)
 		}
 
+		if o.dry {
```

Comment:
Why don't we store the value of `dry` in the logger and use it once, when we output?

Reply:
good idea. I will do it in a followup